### ISSUE TITLE
Tour (Players Rolling Dice from Character Sheet) stops working after step 18

### ISSUE THREAD

**Issue by jeffszusz** [Muttley/foundryvtt-shadowdark#669]
1. The tour of the character sheet gets to step 18, where it shows the character has access to a Greataxe
2. The tour guidance UI then disappears, leaving me looking at the Tour user's inventory
3. Going to the tours menu and attempting to resume the tour does not resume the tour
4. Going to the tours menu and choosing to reset the tour will allow me to take the tour again, but the tour stops working at step 18

Foundry: v11 Stable Patch 13 (build 315)
Shadowdark System: 1.7.3

**Comment by PrototypeESBU** [comment]
@jeffszusz I could not reproduce this error in a new world. It could be that a mod or something is messing with the UI on your world which could cause the tour to fail. Does this happen for you on a new world without any modules enabled?

**Comment by Muttley** [comment]
I'm unable to replicate this one either.

**Comment by Mattrogers13** [comment]
I came here to report the same bug. Should I open a new ticket? I created a new world and had no modules turned on. Here is a screen capture of the experience.

https://github.com/Muttley/foundryvtt-shadowdark/assets/48500042/8d884551-cbe7-46f1-b617-4030eae246fe
